docker-compose.yml

```diff
@@ -11,6 +11,7 @@ services:
       NEO4J_AUTH: none
     volumes:
       - neo4j_data:/data
+      - ./Ingredients/Database/DataSource:/csvfiles  
```

Comment:
I am not sure, do you ever use these volumes? If not it might be a good way off reducing start up times.